sql/core/src/test/scala/org/apache/spark/sql/execution/SQLExecutionSuite.scala

```diff
@@ -129,6 +129,8 @@ class SQLExecutionSuite extends SparkFunSuite {
     df.collect()
 
     assert(df.queryExecution === queryExecution)
+
+    spark.stop()
```

Comment:
Oh, sorry. @hvanhovell . Should I make another followup PR for that to be safer?